easybuild/easyblocks/t/tensorflow.py

```diff
@@ -719,43 +725,50 @@ def build_step(self):
         if LooseVersion(self.version) < LooseVersion('2.0'):
             self.patch_crosstool_files()
 
-        # compose "bazel build" command with all its options...
-        cmd = [
-            self.cfg['prebuildopts'],
-            'bazel',
+        # Options passed to the bazel command
+        self.bazel_opts = [
             '--output_base=%s' % self.output_base_dir,
             '--install_base=%s' % self.install_base_dir,
             '--output_user_root=%s' % self.output_user_root_dir,
-            'build',
         ]
+        jvm_max_memory = self.cfg['jvm_max_memory']
+        if self.cfg['jvm_max_memory']:
```

Comment:
Use the variable you just assigned instead of repeating the self.cfg ?

Reply:
Good catch, thanks!